src/field_region_map.c

```diff
@@ -45,6 +49,12 @@ static void VBCB_FieldUpdateRegionMap(void);
 static void MCB2_FieldUpdateRegionMap(void);
 static void FieldUpdateRegionMap(void);
 static void PrintRegionMapSecName(void);
+static void PrintTitleWindowText(void);
+
+static const u8* const FlyPromptText = COMPOUND_STRING("{R_BUTTON}FLY");
+
+u8 hoennOffset;
+u8 flyOffset;
```

Comment:
why is this outside of a function?

Reply:
i figured it'd be better to set them when the FieldUpdateRegionMap() is called rather than every time the PrintTitleWindowText() function is called to speed up text display

Reply:
Pass them as arguments into `PrintTitleWindowText()` instead then. We don't define variables outside of functions unless they are globals and those don't need to be globals.

Reply:
Also, the function `FieldUpdateRegionMap` is rerun each case which means that they are calculated each time regardless so you aren't gaining any speed improvement with this which means you could just calc it in `PrintTitleWindowText` which will be better for speed because you aren't recalcing each time `FieldUpdateRegionMap` is run.

Reply:
good to know! i thought it was a big loop inside `FieldUpdateRegionMap` lol